packages/now-static-build/src/frameworks.ts

```diff
@@ -413,7 +413,8 @@ const frameworkList: Framework[] = [
   {
     name: 'Jekyll',
     slug: 'jekyll',
-    buildCommand: 'jekyll build',
+    buildCommand: 'rm -f /usr/local/bundle/config && export BUNDLE_PATH=vendor/bundle && jekyll build && bundle clean',
```

Comment:
I don't think we want to do `rm` because that defeats the purpose of the cache. 

I'll have to revisit this when I have more time.

Reply:
Indeed, we can keep `buildCommand: 'jekyll build'`, but in order for the gems to be stored inside the project we have to do some sort of prior configuration.

Someone with access to the build image should take a look at the bundler configuration by running `bundle config`  and add these options there if necessary:
```
rm -f /usr/local/bundle/config
export BUNDLE_PATH=vendor/bundle
```

Reply:
What is the contents of `/usr/local/bundle/config` now?

I think we can avoid modifying the global config and instead modify `runBundleInstall()` to add that environment variable.

Reply:
I also prefer dealing with environment variables instead of messing with the filesystem, however if the file `/usr/local/bundle/config` is present it has a higher precedence than the env var. Just run `bundle config` to see it for yourself.

Reply:
I added `-f` since I don't know whether that file is present or not.

Reply:
The [bundler documentation here](https://bundler.io/man/bundle-config.1.html) states the order or preference.

Reply:
The config file directory might be in a different folder than what I have mentioned earlier.

Reply:
There is no `/usr/local/bundle/config`. The contents of `~/.bundle/config` is:

```
---
BUNDLE_SILENCE_ROOT_WARNING: "1"
```

Reply:
Great, then we won't need to remove anything.
We just have to set the bundler configuration either by setting the environment variables manually or by running `bundle config` with the appropriate arguments.

Reply:
I have changed `runBundleInstall()` to set the required configuration. I think `bundle config` sets the environment variable implicitly. For more information please look at [the documentation](https://bundler.io/man/bundle-config.1.html).

Reply:
Actually the documentation states that `bundle config` writes the configuration to `~/.bundle/config`. Is that alright?
> Executing `bundle config <name> <value>` will set that configuration to the value specified for all bundles executed as the current user. The configuration will be stored in `~/.bundle/config`. If name already is set, name will be overridden and user will be warned.